rskj-core/src/main/java/co/rsk/peg/PegUtilsLegacy.java

```diff
@@ -0,0 +1,393 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.Address;
+import co.rsk.bitcoinj.core.BtcTransaction;
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.bitcoinj.core.Context;
+import co.rsk.bitcoinj.core.ScriptException;
+import co.rsk.bitcoinj.core.TransactionInput;
+import co.rsk.bitcoinj.core.Utils;
+import co.rsk.bitcoinj.script.RedeemScriptParser;
+import co.rsk.bitcoinj.script.RedeemScriptParserFactory;
+import co.rsk.bitcoinj.script.Script;
+import co.rsk.bitcoinj.script.ScriptBuilder;
+import co.rsk.bitcoinj.script.ScriptChunk;
+import co.rsk.bitcoinj.wallet.Wallet;
+import co.rsk.config.BridgeConstants;
+import co.rsk.peg.bitcoin.BitcoinUtils;
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ConsensusRule;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static org.ethereum.config.blockchain.upgrades.ConsensusRule.RSKIP293;
+
+/**
+ * @deprecated Methods included in this class are to be used only prior to the latest HF activation
+ * Use instead methods in {@link co.rsk.peg.PegUtils}
```

Comment:
```suggestion
 * Use instead methods in {@link co.rsk.peg.PegUtils}
```
You Meant To Say - Instead use methods in {@link co.rsk.peg.PegUtils} Or Use methods in {@link... ?